react/features/calendar-sync/functions.any.js

```diff
@@ -90,19 +90,22 @@ function _parseCalendarEntry(event, knownDomains) {
     if (event) {
         const url = _getURLFromEvent(event, knownDomains);
 
-        if (url) {
+        // we only filter events without url on mobile, this is temporary
+        // till we implement event edit on mobile
+        if (url || navigator.product !== 'ReactNative') {
```

Comment:
Another way to detect mobile is to use the browser detection exposed in jitsi js-utils, which I think is accessible through the lib. That said, if this is temporary then I think it's okay to leave in for now.

Reply:
Yep, this whole check about url existence will be removed once mobile implements edit.